nginx/start.sh

```diff
@@ -0,0 +1,11 @@
+#! /bin/bash
+set -e
+
+export IMMICH_WEB_HOST=${IMMICH_WEB_HOST:-immich-web}
```

Comment:
Does this syntax means that if there is no env value of `IMMICH_WEB_HOST` the value `immich-web` will be used instead? So let's say a user not updating their `docker-compose.yml` shouldn't have any problem accessing, correct?

Reply:
Yea, exactly. That sets the default should it not be defined or defined as empty.